components/signup/signup.tsx

```diff
@@ -513,6 +527,7 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
     const handleSubmit = async (e: React.MouseEvent | React.KeyboardEvent) => {
         e.preventDefault();
         trackEvent('signup_email', 'click_create_account', getRoleFromTrackFlow());
+        sendSignUpTelemetryEvents('click_create_account');
```

Comment:
this particular one falls under the `signup` category, I was not sure if from the data analytics perspective for tracking down the first admin data it would be useful when extracting and correlating the data to have this one using the same category.

Reply:
Should we just converge all events in this page to just the `signup` category for consistency? @julmondragon any concerns with that? I see that currently just two events in this component are under `signup_email` category

Reply:
None from my side, this category was ported from the old signup components.
Just make sure the teams using this data are aware of the category change, ie. for the invite flow tracking.

Reply:
@julmondragon any idea how to find the teams that are using this `signup_email` data? I think the best thing to do here is to remove the one I added and make the `signup_email` just to be `signup`

Reply:
@pvev Not sure currently, last time I think @sri-byte was tracking the invite flow.

Reply:
ok, then heads up that we will keep only one event for `click_create_account` under the `signup` category. @marianunez @ifoukarakis @sri-byte